tests/Tests/400__ods_dashboard/413__ods_dashboard_settings_pod_tolerations.robot

```diff
@@ -15,6 +15,9 @@ Test Setting Pod Toleration Via UI
     Wait Until Page Contains    Notebook pod tolerations
     Set Pod Toleration Via UI    TestToleration
     Save Changes In Cluster Settings
```

Comment:
If you look for `Save Changes In Cluster Settings` you'll find at least 2 other places where a Sleep is added after the keyword. Could you add the Sleep inside `Save Changes In Cluster Settings`  and remove the other ones?

Reply:
Are we sure we want the sleep in all cases? 
I'm adding it here simply because the test would fail otherwise as the new setting is not applied quickly enough

Reply:
Done